sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release History
 
 ## 1.7.0-beta.1 (Unreleased)
+### New Features
+- Adding support for `BinaryData` in `AmqpAnnotatedMessage`.
+ 
+### Breaking Changes
+- Updated constructor `AmqpDataMessageBody` to support `BinaryData`.
+- Renamed `AmqpDataBody` to `AmqpDataMessageBody`.
```

Comment:
This is to match .net type name.

Reply:
This list should also include the breaking change to `userId`.